src/files/source/tree.php

```diff
@@ -238,33 +238,72 @@
         'role:sale.master',
     ],
     // STOCK MODULE
-    'role:stock.user' => [
-        'stock.read', 'part.read', 'move.read', 'model.read', 'move.get-directions', 'order.read',
+    'role:part.user' => [
+        'part.read',
     ],
-    'role:stock.admin' => [
-        'role:stock.user',
-        'move.create', 'move.update', 'move.delete',
+    'role:part.manager' => [
+        'role:part.user',
+        'part.create', 'part.update', 'part.delete',
     ],
-    'role:stock.manager' => [
-        'role:stock.user',
+    'role:part.master' => [
         'part.read-all-hierarchy',
-        'part.create', 'part.update', 'part.delete',
+        'role:part.manager',
+    ],
+    'role:move.user' => [
+        'move.read',
+        'move.get-directions',
+    ],
+    'role:move.manager' => [
+        'role:move.user',
         'move.create', 'move.update', 'move.delete',
-        'model.create', 'model.update', 'model.delete',
     ],
-    'role:stock.master' => [
-        'role:stock.manager', 'move.read-all', 'role:order.master',
+    'role:move.master' => [
+        'role:move.manager',
+        'move.read-all',
+    ],
+    'role:model.user' => [
+        'model.read',
+    ],
+    'role:model.manager' => [
+        'role:model.user',
+        'model.create', 'model.update',
+    ],
+    'role:model.master' => [
+        'role:model.manager', 'model.delete',
```

Comment:
What use case should cover the `model.master` role? It seems redundant to me: clients' managers will never create or update models, and internal managers can delete only models without parts (because of DB constraints).